specification.markdown

```diff
@@ -160,7 +160,7 @@ Octet Length | Possible Value Range
 7            | 0 to 2^49-2
 8            | 0 to 2^56-2
 
-If the length of `Element Data` equals `2^(n*7)-1` then the octet length of the `Element Data Size` MUST be at least `n+1`. This rule prevents an `Element Data Size` from being expressed as a reserved value. The following table clarifies this rule by showing a valid and invalid expression of an `Element Data Size` with a `VINT_DATA` of 127 (which is equal to 2^(1*7)-1) and 16,383 (which is equal to 2^(2*7)-1).
+If the length of Element Data equals `2^(n*7)-1 then the octet length of the Element Data Size MUST be at least n+1`. This rule prevents an Element Data Size from being expressed as a reserved value. The following table clarifies this rule by showing a valid and invalid expression of an Element Data Size with a VINT_DATA of 127 (which is equal to 2^(1*7)-1) and 16,383 (which is equal to 2^(2*7)-1).
```

Comment:
The backticked block here includes two forumulae and the statement in between.

Reply:
OK I will merge your patch and then update this one.

Reply:
There's still two backticks left in here which join together two formula and text into a part that shouldn't be singularly encapsulated.

Reply:
Fixed